### DISCUSSION TITLE
Is it possible to use local notifications?

### DISCUSSION THREAD

**Discussion by ComptonAlvaro** [dotnet/maui#14962] [Q&A]
### Description

When an event happens in the application, I would like to create a notification to the user.

All the examples that I have seen are push notifications, that are notifications that I have receive from internet, but I would like to know if it is possible to do it without internet, just from application to notifications.

If there are some example o documentation that explain how to do it, it would be a very good start ponit.

Thanks.

### Public API Changes

There is no API changes

### Intended Use-Case

When an application runs in the background, I would like to get a notificaction from the application if something has to be notificate to the user.

**Comment by angelru** [comment]
@ComptonAlvaro  https://github.com/thudugala/Plugin.LocalNotification
